java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture10/tasks/_1/Menu.java

```diff
@@ -0,0 +1,16 @@
+package com.zoolatech.lecture10.tasks._1;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+final class Menu {
+    public static void menu() throws IOException {
```

Comment:
Please do not use static here. just new Menu().menu(). It's not a mistake but static methods are always a problem for testing.